Re-list Intercom for mobile destination plugins (#6136)
segmentio/segment-docs: src/connections/destinations/catalog/intercom/index.md

id: 54521fd725e721e32a72eec6
private: true
maintenance: true
maintenance-content: This destination should only be used for Mobile connections. New versions of the destination are available for browser and server connections. See [Intercom Cloud Mode (Actions)](/docs/connections/destinations/catalog/actions-intercom-cloud/) and [Intercom Web (Actions)](/docs/connections/destinations/catalog/actions-intercom-web/) for more information.
---

## Getting Started

1.  From the Segment Destinations page click **Add Destination**.
2.  Search for "Intercom" and select it in the results that appear.

> 3.  Choose which Source to connect Intercom to, this shoud only be a Kotlin or Swift Mobile source.

Reviewer: ```suggestion
3.  Choose a Kotlin or Swift Mobile source to connect to Intercom.
```

4.  Authorize your Intercom account in Segment and select the Intercom Account to sync with Segment.
5. [Find your "App ID" in the Intercom UI](https://docs.intercom.com/faqs-and-troubleshooting/getting-set-up/where-can-i-find-my-app-id){:target="_blank"} or by navigating to the Gear Menu and clicking on "App Settings" followed by "API Keys". It should look something like `9iefb489`.

### Mobile

#### Kotlin

> To find implementation details for our Kotlin Intercom Destination Plugin, please review our documentation [here](/docs/connections/sources/catalog/libraries/mobile/kotlin-android/destination-plugins/intercom-kotlin-android/).

Reviewer: ```suggestion
To find implementation details for Segment's Kotlin Intercom Destination Plugin, please review the [Intercom plugin documentation](/docs/connections/sources/catalog/libraries/mobile/kotlin-android/destination-plugins/intercom-kotlin-android/).
```

#### Swift

> To find implementation details for our Swift Intercom Destination Plugin, please review our documentation [here](/docs/connections/sources/catalog/libraries/mobile/apple/destination-plugins/intercom-swift/).

Reviewer: ```suggestion
To find implementation details for Segment's Swift Intercom Destination Plugin, please review the [Intercom plugin documentation](/docs/connections/sources/catalog/libraries/mobile/apple/destination-plugins/intercom-swift/).
```
